packages/ui-components/src/Tooltip/Tooltip.scss

```diff
@@ -0,0 +1,189 @@
+// @import "../styles/tokens.scss";
+
+
+//
+// Tooltips
+// --------------------------------------------------
+
+$font-size-base         : 12px;
+$line-height-base       : 1.5;
+$border-radius-base     : 3px;
+$overlay-shadow         : 0 0 4px rgba(0, 0, 0, 0.17);
+//** Tooltip text color
+$tooltip-color: #333333;
+//** Tooltip background color
+$tooltip-bg: #ffffff;
+$tooltip-opacity: 1;
+
+$tooltip-border-width: 1px;
+$tooltip-border-color: #b1b1b1;
+$tooltip-shadow-width: 1px;
+
+$tooltip-arrow-width: 6px;
+$tooltip-arrow-color: $tooltip-border-color;
+$tooltip-arrow-inner-color: $tooltip-bg;
```

Comment:
Font sizes, colors, border radii, shadows, etc. should definitely be tokenized and consumed as opposed to defined here.  Since we only have a small number of things tokenized right now, I'd recommend subbing in token values wherever possible, adding a TODO above these styles re: tokenization, and filing an issue to create tokens for anything that might be missing in tokens that is being defined here or elsewhere.